modules/logging-loki-storage-aws.adoc

```diff
@@ -19,18 +21,21 @@
 +
 [source,terminal,subs="+quotes"]
 ----
-$ oc create secret generic logging-loki-aws \
+$ oc create secret generic logging-loki-aws \ #<1>
   --from-literal=bucketnames="<bucket_name>" \
   --from-literal=endpoint="<aws_bucket_endpoint>" \
   --from-literal=access_key_id="<aws_access_key_id>" \
   --from-literal=access_key_secret="<aws_access_key_secret>" \
   --from-literal=region="<aws_region_of_your_bucket>"
+  --from-literal=forcepathstyle="true" # <2>
 ----
+<1> `logging-loki-aws` is the secret name.
```

Comment:
Being picky. Maybe change to the name of the secret? When I first read it, I thought the name was a secret.  

```suggestion
<1> `logging-loki-aws` is the name of the secret.
```